src/main/content/antora_ui/src/js/06-javadocs.js

```diff
@@ -248,6 +249,7 @@ function addLeftFrameScrollListener(frameToListen, frameElementToListen) {
   var frame = $("#javadoc_container")
     .contents()
     .find(frameToListen);
+  if(frame.length === 0) return;
```

Comment:
Best practice to always use brackets
```suggestion
  if(frame.length === 0) {
       return;
   }
```